modules/cluster-logging-collector-external.adoc

```diff
@@ -64,26 +71,71 @@ Additionally, you can add any certificates required by your configuration to a s
 </store> 
 ----
 
+.Sample `secure-forward` configmap based on the configuration
+
+[source,yaml]
+----
+apiVersion: v1
+data:
+ secure-forward.conf: "<store>
+     \ @type forward
+     \ <security>
+     \   self_hostname ${hostname} # ${hostname} is a placeholder.
+     \   shared_key \"fluent-receiver\"
+     \ </security>
+     \ transport tls
+     \ tls_verify_hostname false           # Set false to ignore server cert hostname.
+     \ tls_cert_path '/etc/ocp-forward/ca-bundle.crt'
+     \ <buffer>
+     \   @type file
+     \   path '/var/lib/fluentd/secureforwardlegacy'
+     \   queued_chunks_limit_size \"#{ENV['BUFFER_QUEUE_LIMIT'] || '1024' }\"
+     \   chunk_limit_size \"#{ENV['BUFFER_SIZE_LIMIT'] || '1m' }\"
+     \   flush_interval \"#{ENV['FORWARD_FLUSH_INTERVAL'] || '5s'}\"
+     \   flush_at_shutdown \"#{ENV['FLUSH_AT_SHUTDOWN'] || 'false'}\"
+     \   flush_thread_count \"#{ENV['FLUSH_THREAD_COUNT'] || 2}\"
+     \   retry_max_interval \"#{ENV['FORWARD_RETRY_WAIT'] || '300'}\"
+     \   retry_forever true
+     \   # the systemd journald 0.0.8 input plugin will just throw away records if the buffer
+     \   # queue limit is hit - 'block' will halt further reads and keep retrying to flush the
+     \   # buffer to the remote - default is 'exception' because in_tail handles that case
+     \   overflow_action \"#{ENV['BUFFER_QUEUE_FULL_ACTION'] || 'exception'}\"
+     \ </buffer>
+     \ <server>
+     \   host fluent-receiver.openshift-logging.svc  # or IP
+     \   port 24224
+     \ </server>
+     </store>"
+kind: ConfigMap
+metadata:
+  creationTimestamp: "2020-01-15T18:56:04Z"
+  name: secure-forward
+  namespace: openshift-logging
+  resourceVersion: "19148"
+  selfLink: /api/v1/namespaces/openshift-logging/configmaps/secure-forward
+  uid: 6fd83202-93ab-d851b1d0f3e8
+----
+
 .Procedure
 
 To configure the *out_forward* plug-in:
 
-. Create a ConfigMap called `secure-forward` in the openshift-logging namespace:
+. Create a configuration file named `secure-forward.conf` for the *out_forward* parameters: 
 +
 .. Configure the secrets and TLS information:
 +
 [source,yaml]
 ----
  <store>
-  @type secure_forward
+  @type forward
 
   self_hostname ${hostname} <1>
   shared_key <SECRET_STRING> <2>
 
   transport tls <3>
 
   tls_verify_hostname true <4>
-  tls_cert_path /etc/ocp-forward/ca_cert.pem <5>
+  tls_cert_path <path to file> <5>
```

Comment:
For _\<variable name\>_ we seem to have the most usage by far of _, followed by -, with a handful of files using a space. So I'd advocate for:

s/path to file/path_to_file/